islandora_transforms/slurp_all_MODS_to_solr.xslt

```diff
@@ -1,245 +1,174 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Basic MODS -->
-<xsl:stylesheet version="1.0"
-  xmlns:java="http://xml.apache.org/xalan/java"
-  xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
-  xmlns:foxml="info:fedora/fedora-system:def/foxml#"
-  xmlns:mods="http://www.loc.gov/mods/v3"
-     exclude-result-prefixes="mods java">
-  <!-- <xsl:include href="/usr/local/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/config/index/FgsIndex/islandora_transforms/library/xslt-date-template.xslt"/>-->
-  <xsl:include href="/usr/local/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/library/xslt-date-template.xslt"/>
-  <!-- <xsl:include href="/usr/local/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/config/index/FgsIndex/islandora_transforms/manuscript_finding_aid.xslt"/> -->
-  <xsl:include href="/usr/local/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/manuscript_finding_aid.xslt"/>
-  <!-- HashSet to track single-valued fields. -->
-  <xsl:variable name="single_valued_hashset" select="java:java.util.HashSet.new()"/>
-
-  <xsl:template match="foxml:datastream[@ID='MODS']/foxml:datastreamVersion[last()]" name="index_MODS">
```

Comment:
@nigelgbanks: In this case they are losing some of the functionality that was added since the original fork such as sortable string fields and single valued fields.